skbio/util/_decorator.py

```diff
@@ -13,6 +13,7 @@
 import decorator
 
 from ._exception import OverrideError
+from ._warning import DeprecationWarning as SkbioDeprecationWarning
```

Comment:
Wait never mind, I subclass `DeprecationWarning` for that very reason. So it wouldn't change anything for the user.